src/system.h

```diff
@@ -297,7 +297,7 @@ struct System {
                     case 'I':
                     case 'J': {
                         char iti = *buf;
-                        wxBitmapType imt = imagetypes[iti].first;
+                        wxBitmapType imt = imagetypes.at(iti).first;

```

Comment:
`at` should only be used if you're 100% sure the value is correct, since it will throw an exception. Here, `find` would be more appropriate.

Reply:
Thanks for your feedback. I changed it accordingly.